compile/transform.ts

```diff
@@ -96,7 +96,7 @@ export const transformMouldToReactComponent = (mould: Mould): string => {
         } else if (type === 'Mould') {
             compType = ensureComponentName(props['__mouldName'])
         } else {
-            const transform = m.transforms[type]
+            const transform = transforms[type]!
```

Comment:
I think, we can allow ourselves to use the non-null assertion operator, because we need to exit the compilation in any case if some transform doesn't exist. We can't check in in advance

Reply:
Sounds good!